### ISSUE TITLE
macOS Support

### ISSUE THREAD

**Issue by brianmichel** [movingparts-io/Pow#18]
Pow seems really rad, but I'd really like to use it in some macOS applications! This is just an issue to start the discussion around this potentially rather large feature request.

**Comment by robb** [comment]
Hi Brian, there's macOS support in Pow for everything but the most recent `SoundEffect`s, even if it's poorly advertised.

I realize the `Package.swift` doesn't specify but we support macOS 12.6 and up, will try to get that fixed and add an macOS target to the Example app.

Let me know if there's anything else I can help with in the meantime!

**Comment by robb** [comment]
Here's a quick first pass at a macOS version of the Pow Example app: https://github.com/movingparts-io/Pow-Examples/pull/2.

**Comment by brianmichel** [comment]
Thanks Robb, I’ll take a peak, please consider this issue closed for the
time being!

**Comment by robb** [comment]
Cool, feel free to reopen if you have any questions!

**Comment by robb** [comment]
See #20

**Comment by mergesort** [comment]
I figured it would be better to add to this issue rather than create a new issue. I was wondering if it makes more sense in a cross-platform codebase to make code like this no-op rather than throwing an error, `Type 'AnyChangeEffect' has no member 'feedbackHapticSelection'`.
```swift
.changeEffect(.feedbackHapticSelection, value: self.isShuffling, isEnabled: self.isShuffling)
``` 

I'm porting an over to the Mac and while it's possible for me to add conditional compilation flags to solve this, there are a lot of call-sites (as I love Pow) and I figured it might be worth asking if the better approach was to have the API work but just not do anything on the Mac.
